.travis.yml

```diff
@@ -0,0 +1,27 @@
+before_script:
+  - ruby --version
+
+env:
+  global:
+    - APM_TEST_PACKAGES="language-haml linter"
+
+  matrix:
+    - ATOM_CHANNEL=stable
+    - ATOM_CHANNEL=beta
+
+language: ruby
+
+notifications:
+  email:
+    on_success: never
+    on_failure: change
+
+os:
```

Comment:
OS X only officially supports Objective-C, but it doesn't _prevent_ you from installing anything else.